src/expressions/call-expr.md

```diff
@@ -1,5 +1,12 @@
 # Call expressions
 
+> **<sup>Syntax</sup>**  
+> _CallExpression_ :  
+> &nbsp;&nbsp; [_Expression_] `(` _CallParams_<sup>?</sup> `)`  
+>   
+> _CallParams_ :  
+> &nbsp;&nbsp; [_Expression_]&nbsp;( `,` ([_Expression_]) )<sup>\*</sup> `,`<sup>?</sup>  
```

Comment:
The parenthesis around the second `Expression` are extraneous.

Reply:
Corrected, thanks.